### PR TITLE
Classifications via properties files

### PR DESCRIPTION
**Pull request by bhreinb**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -170,7 +170,7 @@
         <dependency>
             <groupId>net.masterthought</groupId>
             <artifactId>cucumber-reporting</artifactId>
-            <version>3.15.0</version>
+            <version>3.16.0</version>
```
Comment (damianszczepanik) [inline pom.xml]: are you working on the most recent master version ? 
https://github.com/damianszczepanik/cucumber-sandwich/blob/master/pom.xml#L173

Comment (bhreinb) [reply pom.xml]: Yeah 3.16 is the most recent edition.
https://github.com/damianszczepanik/cucumber-reporting/releases

Comment (damianszczepanik) [reply pom.xml]: So why did you need to upgrade from older version 3.15?

**Comment by damianszczepanik** [comment]
It does. The real question was why did you not rebase the code before sending pull request. Do this then send changes again

**Comment by bhreinb** [comment]
Apologies, I didn't do a rebase as the only file that changed within the 10 days from your commit was

https://github.com/damianszczepanik/cucumber-sandwich/commits/master/pom.xml

which was upgrading the dependency version & updating the version of the library. I have aligned that manually so hopefully the PR is good for you.

**Comment by damianszczepanik** [comment]
Closing the PR as the rebase is still not performed properly

**Comment by bhreinb** [comment]
I submitted the PR with the changes...pity their wasn't any flexibility on your part on accepting it.

**Comment by damianszczepanik** [comment]
It was not ready to be merged and hasn't been updated for over 7 months.

**Comment by bhreinb** [comment]
I disagree 7 months ago it was ready. Correct it hasn't been touched since.

**Pull request closed without merging**